### DISCUSSION TITLE
Blueretro retroscaler ps2 adapter Light Issue

### DISCUSSION THREAD

**Discussion by TRSE01** [darthcloud/BlueRetro#1136] [Q&A]
Hello, 
I bought a Blueretro ps2 adapter so that I could connect my ps3 controllers to the ps2. I wanted to update the firmware for my adapter but all I got was the lights for the blueretro being solid and not connecting. I followed the instructions from these links:

https://www.retroscaler.com/wp-content/uploads/2023/06/Blueretro-PS1PS2-manual.pdf
https://github.com/RetroScaler/Bluereteo-PS?tab=readme-ov-file

I then noticed that another person had this same issue so their solution was to use this link:

https://github.com/darthcloud/BlueRetro/wiki/Flashing-firmware-Windows-10

I did the tutorial but that did not solve the issue for the firmware update and the solid lights.

I am new to this and it would be much appreciated if anyone can help me what am I doing wrong. 

Also side note:
I tried resetting it and followed the instructions to do it but it does not work.
I bought this blueretro ps2 adapter on Amazon.
https://www.amazon.com/dp/B0C5M474CL?ref=ppx_yo2ov_dt_b_fed_asin_title&th=1

**Comment by lilkuz2005** [comment]
When I updated the firmware on my RetroScaler blueretro ps2 adapter I powered it with a usb cable connected to my pc, I made sure no gamepads were connected to the blueretro and I opened the blueretro.io page and went to system management and paired the blueretro to my pc via bluetooth.
